crates/bevy_render/src/lib.rs

```diff
@@ -107,6 +107,10 @@ pub struct RenderApp;
 #[derive(Default)]
 struct ScratchRenderWorld(World);
 
+/// The number of entities used in the render world in the previous frame.
+#[derive(Default)]
```

Comment:
I think it may make sense to initialize this at something low but not tiny, like perhaps 100?